front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx

```diff
@@ -0,0 +1,172 @@
+import { Fab, Grid, Paper, TextField, Typography } from "@material-ui/core";
+import ClearIcon from "@material-ui/icons/Clear";
+import DoneIcon from "@material-ui/icons/Done";
+import EditIcon from "@material-ui/icons/Edit";
+import React, { Fragment } from "react";
+import { updateOrganizationNameDatabase } from "../../../Scripts/firebaseOrganizationUpdates";
+import { PageProps } from "../../Pages";
+
+declare interface ViewEditOrganizationInfoProps {
+  currentOrganizationData: any;
+  setNotification: any;
+  handleLoadOrganizationData: any;
+  setLoadingMessage: any;
+  classes: any;
+}
+
+const ViewEditOrganizationName: React.FunctionComponent<ViewEditOrganizationInfoProps> = ({
+  currentOrganizationData,
+  setNotification,
+  handleLoadOrganizationData,
+  setLoadingMessage,
+  classes,
+}) => {
+  const [organizationName, setOrganizationName] = React.useState<string>(
+    currentOrganizationData?.organizationName
+      ? currentOrganizationData.organizationName
+      : ""
+  );
+  const [editing, setEditing] = React.useState(!organizationName);
+
+  const startEditing = () => {
+    setEditing(true);
+  };
+
+  const cancelEditing = () => {
+    setEditing(false);
+  };
+
+  const cancelEditingOrganizationName = () => {
+    cancelEditing();
+    setOrganizationName(
+      currentOrganizationData?.organizationName
+        ? currentOrganizationData.organizationName
+        : ""
+    );
+  };
+
+  const saveOrganizationName = () => {
+    if (currentOrganizationData) {
+      setLoadingMessage("Updating Organization Name...");
+      updateOrganizationNameDatabase(
+        currentOrganizationData.userId,
+        organizationName
+      )
+        .then((value: any) => {
```

Comment:
You shouldn't need to specify any... See if you can take the any out.

Reply:
Done